app/routers/api/migrations.py

```
@@ -0,0 +1,22 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from app.database import get_db
+from app.services.relocation_manager import relocation_manager
+
+router = APIRouter(prefix="/api/v1/migrations", tags=["migrations"])
+logger = logging.getLogger(__name__)
+
+
+@router.get("/active", response_model=List[Dict[str, Any]])
+def get_active_migrations(db: Session = Depends(get_db)): # NOSONAR
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)